notebooks/002-datamodules/002-btech.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>I'm missing a short explanation here explaining what's in the dictionary for the test split, similar to above for the train split.</p><p> </p><p>Also, is there a reason why the test split includes the <code>image_path</code> in the dictionary, but the train split doesn't? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>